test/uk/gov/hmrc/nisp/views/TermsAndConditionsViewSpec.scala

```diff
@@ -0,0 +1,88 @@
+import play.api.mvc.{AnyContent, Request}
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+import uk.gov.hmrc.http.HttpPost
+import uk.gov.hmrc.nisp.config.ApplicationConfig
+import uk.gov.hmrc.nisp.config.wiring.{NispFormPartialRetriever, WSHttp}
+import uk.gov.hmrc.nisp.controllers.FeedbackController
+import uk.gov.hmrc.nisp.controllers.auth.NispAuthedUser
+import uk.gov.hmrc.nisp.fixtures.NispAuthedUserFixture
+import uk.gov.hmrc.nisp.helpers._
+import uk.gov.hmrc.nisp.utils.{Constants, FakeApplicationConfig, MockTemplateRenderer}
+import uk.gov.hmrc.play.partials.{CachedStaticHtmlPartialRetriever, FormPartialRetriever}
+import uk.gov.hmrc.renderer.TemplateRenderer
+
+class TermsAndConditionsViewSpec extends HtmlSpec with MockitoSugar {
+
+  val fakeRequest = FakeRequest("GET", "/")
+
+  val mockHttp = mock[WSHttp]
+
+  val testFeedbackController = new FeedbackController {
+    override implicit val cachedStaticHtmlPartialRetriever: CachedStaticHtmlPartialRetriever = MockCachedStaticHtmlPartialRetriever
+    override implicit val formPartialRetriever: FormPartialRetriever = MockFormPartialRetriever
+
+    override implicit val templateRenderer: TemplateRenderer = MockTemplateRenderer
+
+    override def httpPost: HttpPost = mockHttp
+
+    override def localSubmitUrl(implicit request: Request[AnyContent]): String = ""
+
+    override def contactFormReferer(implicit request: Request[AnyContent]): String = request.headers.get(REFERER).getOrElse("")
+
+    override val applicationConfig: ApplicationConfig = new FakeApplicationConfig {}
+
+  }
+
+  implicit val cachedStaticHtmlPartialRetriever = MockCachedStaticHtmlPartialRetriever
+  implicit val formPartialRetriever: uk.gov.hmrc.play.partials.FormPartialRetriever = NispFormPartialRetriever
+  implicit val templateRenderer: TemplateRenderer = MockTemplateRenderer
+  implicit val user: NispAuthedUser = NispAuthedUserFixture.user(TestAccountBuilder.regularNino)
+
+  val feedbackFrontendUrl: String = "/foo"
+  lazy val html = uk.gov.hmrc.nisp.views.html.termsAndConditions(true)
+  lazy val source = asDocument(contentAsString(html))
+
+  "TermsAndConditions" should {
+
+    "assert correct page title" in {
+      val title = source.title()
+      val expected = messages("nisp.tandcs.title") + Constants.titleSplitter +
+                     messages("nisp.title.extension") + Constants.titleSplitter + messages("nisp.gov-uk")
+      title must include(expected)
+    }
+
+    "assert correct heading title on page" in {
+      val title = source.getElementsByTag("h1").get(0).toString
+      val messageKey = "nisp.tandcs.title"
+      val expected = "> " + messages(messageKey) + " <"
+      title must include(expected)
+    }
+
+    "assert correct heading level 2 on page" in {
+      val title = source.getElementsByTag("h2").get(0).toString
+      val messageKey = "nisp.tandcs.heading"
+      val expected = ">" + messages(messageKey) + "<"
+      title must include(expected)
+    }
```

Comment:
Given that there is a h1 and h2 on this page. Is there any other content that could also be captured in this spec?